internal/dependencymanager/add.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dependencymanager
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/flowkit"
+	"github.com/onflow/flow-cli/flowkit/output"
+	"github.com/onflow/flow-cli/internal/command"
+)
+
+type addFlagsCollection struct {
+	name string `default:"" flag:"name" info:"Name of the dependency"`
+}
+
+var addFlags = addFlagsCollection{}
+
+var addCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:   "add",
```

Comment:
add placeholder to tell what to provide for an argument, check other commands for syntax, I believe something like: 
```
add <name of the dependency>
```